channels/channelmapper_test.go

```diff
@@ -9,595 +9,650 @@
 package channels
 
 import (
+	"encoding/json"
 	"fmt"
+	"strings"
 	"testing"
 
+	"github.com/couchbase/sg-bucket/js"
 	"github.com/couchbase/sync_gateway/base"
-	"github.com/robertkrimen/otto"
-	"github.com/robertkrimen/otto/underscore"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
+	// Docs: https://pkg.go.dev/github.com/snej/v8go
 )
 
-func init() {
-	underscore.Disable() // It really slows down unit tests (by making otto.New take a lot longer)
-}
-
```

Comment:
we don't want to drop this when we still support otto, move to `main_test.go` is fine

Reply:
See earlier comment on the other instance of this; tl;dr I took it out because it's not slow anymore.